### PR TITLE
Update ReadMe

### LINKED ISSUE #18: Update ReadMe & Wiki
## Description

To facilitate usage and installation there should be a read me file that can guide a user through work.

## Possible Implementation

In depth installation instructions can live in the Wiki while the readme can contain a "quick start" set of instructions

## Out of scope

N/A
## Requirements / Restrictions

Should cover as many operating systems as possible

### PR DESCRIPTION
**Pull request by m-triassi**
## Description
Closes #18
Adds a relevant ReadMe file to the project, outlining  basic information, installation and usage instructions, as well as Contributing and Licensing information.

Additionally the [wiki](https://github.com/DnD-Montreal/session-tome/wiki) was filled out along side this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,92 @@
...
-- **[CMS Max](https://www.cmsmax.com/)**
-- **[WebReinvent](https://webreinvent.com/?utm_source=laravel&utm_medium=github&utm_campaign=patreon-sponsors)**
+
+## Usage
+
+### Docker Compose
+Once all the dependencies have been installed and set-up is complete, from the project root simply run:
+
+```shell
+docker compose up -d
```
Comment (pascwhale) [inline README.md:56]:
```suggestion
docker network create web-public
docker compose up -d
```

**Review by pascwhale — CHANGES_REQUESTED** [review]
A user running the `docker-compose up` command should first create a docker network.

File: README.md (hunk 1 above)
Comment (m-triassi) [reply README.md:56]: Ah missed that! committing

**Review by pascwhale — APPROVED** [review]
Great, everything looks good!

**Pull request merged**